pkgs/development/python-modules/torch/default.nix

```diff
@@ -4,6 +4,10 @@
   MPISupport ? false, mpi,
   buildDocs ? false,
 
+  # tests.gpu:
+  callPackage,
+  torch,
```

Comment:
Nix allows this? Passing `torch` to the derivation that makes `torch`?

Reply:
Sure, because it's not used by torch itself, only by the passthru

Reply:
Sure, because it's not used by torch itself, only by the passthru